src/resources/scripts/toCoco.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+import numpy
+from shapely.geometry import Point
+from shapely.affinity import scale,rotate
+import warnings
+import sys
+import math
+import json
+import ast
+
+# target_file = sys.argv[2]
+# csv = sys.argv[3]
+
+# the path to the CSV file
+paths = sys.argv[3:]
+# json (output) path #ignore for now
+save_json_path = sys.argv[2]
+
+
+def check_shape(row):
+    # only polygons are accepted
+    shape = row.shape_name
+    flag_shape = 1
+    if not(shape == "LineString" or shape == "Polygon" or shape == "Rectangle" or shape=="Circle" or shape=="Ellipse") :
+        warnings.warn('The shape %s is not supported !' %(shape))
+        flag_shape = 0
+    return flag_shape
```

Comment:
Shouldn't this return either `True` or `False` (since it is used in an `if` below)?

Reply:
This is exactly the same (there is a very dirty case only possible in python2 where this is not the case) but I change it for readability.